build.gradle.kts

```diff
@@ -1,10 +1,10 @@
 plugins {
     id("java")
-    id("org.jetbrains.intellij") version "1.10.1"
+    id("org.jetbrains.intellij") version "1.13.3"
```

Comment:
The plugin version upgrade so fast, it maybe need to test. So i want keep using 1.10.1 until it can't use.